mariadb/templates/pvc.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.usePersistence }}
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: {{ .Values.dataVolume.storageClass | quote }}
```

Comment:
According to https://github.com/kubernetes/kubernetes/tree/master/examples/experimental/persistent-volume-provisioning#user-provisioning-requests, this should be `volume.beta.kubernetes.io/storage-class`?